cmd/ocitools/validate.go

```diff
@@ -84,7 +84,7 @@ var bundleValidateCommand = cli.Command{
 			return fmt.Errorf("The root path %q is not a directory.", rootfsPath)
 		}
 
-		hostCheck := context.Bool("host-specific")
+		hostCheck := hostSpecific || context.Bool("host-specific")
```

Comment:
Ah, [here][1]'s an example of accessing a global option.

[1]: https://github.com/opencontainers/ocitools/pull/138/files#diff-46eb56a79aaa28b90fd1b953e23bc8c9R40